.gitignore

```diff
@@ -9,3 +9,4 @@ schema.graphql
 comet-lang
 /api/uploads
 lang
+lib
```

Comment:
Please move this to the respective .gitignore of the package

Reply:
moved in 8dfd38991f975e0390502a413279292dc8fdb117